test/CMakeLists.txt

```
@@ -1,3 +1,32 @@
+function(COMBINE_ADD_GTEST test_suite)
+  cmake_parse_arguments(ARG
+    ""
+    "TIMEOUT;FAILREGEX"
+    "COPY_TO_BUILDDIR;LIBRARIES;LABELS;INCLUDE_DIRS;ENVIRONMENT" ${ARGN})
+
+  ROOT_GET_SOURCES(source_files . ${ARG_UNPARSED_ARGUMENTS})
+  ROOT_EXECUTABLE(${test_suite} ${source_files} LIBRARIES ${ARG_LIBRARIES})
+  target_link_libraries(${test_suite} PRIVATE GTest::gtest GTest::gmock GTest::gtest_main GTest::gmock_main)
```

```suggestion
  ROOT_EXECUTABLE(${test_suite} ${source_files}
      LIBRARIES ${ARG_LIBRARIES} GTest::gtest GTest::gmock GTest::gtest_main GTest::gmock_main
  )
```
@maxgalli, I think this is the way to fix the build error on older ROOT versions